test/system/workflow_executions_test.rb

```diff
@@ -16,26 +16,93 @@ class WorkflowExecutionsTest < ApplicationSystemTestCase
     assert_selector 'table#workflow_executions tbody tr', count: 12
   end
 
+  test 'should sort a list of workflow executions' do
```

Comment:
Can I get you to add in testing for the pagination as well? You will need to add more workflow executions to the fixtures. Take a look at `test/system/groups/members_test.rb` -> `test 'can see the list of group members'` to see how we've tested it in the past

Reply:
4fb2e260d329cf6650ddf1d01e9e26f7e1b2e204